SportsHub/SportsHub.Application/Services/ArticleService.cs

```diff
@@ -0,0 +1,35 @@
+﻿using SportsHub.Domain.Models;
+using SportsHub.Domain.UOW;
+
+namespace SportsHub.AppService.Services
+{
+    public class ArticleService : IArticleService
+    {
+        private readonly IUnitOfWork unitOfWork;
+
+        public ArticleService(IUnitOfWork unitOfWork)
+        {
+            this.unitOfWork = unitOfWork;
+        }
+
+        public async Task<Article?> GetByTitleAsync(string title)
+        {
+            return await unitOfWork.ArticleRepository.GetByTitleAsync(title);
+        }
+
+        public async Task<IEnumerable<Article>> GetByAuthorAsync(string author)
+        {
+            return await unitOfWork.ArticleRepository.GetByAuthorAsync(author);
```

Comment:
do we use all these methods?

Reply:
There were no requirements when it came to methods - I wrote several so we have something to work with

Reply:
I am not sure that it is a good idea, code will hard to support. It is good that you have done more work, but move it to the separate Branch or stash it, please

Reply:
Should I stash/branch from everywhere - interfaces, repository, service?

Reply:
Everything that is not used is now moved to a local branch.